account_invoice_refund_line_selection/wizards/account_move_reversal_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_account_invoice_refund_lines" model="ir.ui.view">
+        <field name="name">account.move.reversal.lines.form</field>
+        <field name="model">account.move.reversal</field>
+        <field name="inherit_id" ref="account.view_account_move_reversal" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='reason']" position="before">
+                <field name="refund_lines" />
+                <div invisible="not refund_lines" class="oe_grey" colspan="2">
+                    Use this option if you want to refund only some specific lines in an invoice.
+                </div>
+            </xpath>
+            <xpath expr="//footer" position="before">
+                <notebook invisible="not refund_lines">
+                    <page string="Invoice Lines">
+                        <field name="selectable_invoice_lines_ids" invisible="1" />
```

Comment:
```suggestion
```

Normally not necessary according to https://github.com/odoo/odoo/pull/137031

Reply:
@imlopes Can you please attend this comment? :pray: